cross/go2rtc/Makefile

```diff
@@ -0,0 +1,34 @@
+PKG_NAME = go2rtc
+PKG_VERS = 1.6.2
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/AlexxIT/go2rtc/archive
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+EXTRACT_PATH = $(WORK_DIR)/src/github.com/$(PKG_NAME)
+PKG_DIR = src/github.com/$(PKG_NAME)/$(PKG_NAME)
+
+BUILD_DEPENDS = native/go
+
+HOMEPAGE = https://github.com/AlexxIT/go2rtc/
+COMMENT  = Ultimate camera streaming application with support RTSP, RTMP, HTTP-FLV, WebRTC, MSE, HLS, MP4, MJPEG, HomeKit, FFmpeg, etc.
+LICENSE  = MIT
+
+POST_EXTRACT_TARGET = go2rtc_post_extract
+
+GO_SRC_DIR = $(EXTRACT_PATH)/$(PKG_NAME)
+GO_BIN_DIR = $(GO_SRC_DIR)/$(PKG_NAME)
+
+ENV += CGO_ENABLED=0
+
```

Comment:
This definition should not be required.

We already have a Makefile variable `CGO_ENABLED` that is used by the framework (spksrc.cross-go.mk) and it defaults to `CGO_ENABLED = 0`. (see `mk/spksrc.cross-go-env.mk`)
```suggestion
```